### PR TITLE
More release day changes

### PR DESCRIPTION
**Pull request by dokun1**
- more code cleanup (please please please no more force-unwrapping optionals!!!)
- swapped positions of camera and checklist icons for much benefit glorious state of UX

### CODE REVIEW

File: iOS/rainbow/Controller/MainTabBarController.swift [hunk 1]
```diff
@@ -60,11 +60,11 @@ class MainTabBarController: UITabBarController {
             
             ScoreEntry.ServerCalls.save(entry: newEntry, completion: { entry, error in
                 if error != nil {
-                    SVProgressHUD.showError(withStatus: "Could not save user")
+                    SVProgressHUD.showError(withStatus: "You'll need an internet connection to get on the leaderboard!")
```
Comment (sanjeevghimire) [inline iOS/rainbow/Controller/MainTabBarController.swift]: could this error message be like: "Error getting response from server. Please try again"

File: iOS/rainbow/Controller/MainTabBarController.swift [hunk 2]
```diff
@@ -73,7 +73,7 @@ class MainTabBarController: UITabBarController {
                         do {
                             try ScoreEntry.ClientPersistence.save(entry: newEntry)
                         } catch let saveError {
-                            SVProgressHUD.showError(withStatus: "Could not save user")
+                            SVProgressHUD.showError(withStatus: "You'll need an internet connection to get on the leaderboard!")
```
Comment (sanjeevghimire) [inline iOS/rainbow/Controller/MainTabBarController.swift]: this is saving in core data. so error message could be different

Comment (dokun1) [reply iOS/rainbow/Controller/MainTabBarController.swift]: Good catch.

File: iOS/rainbow/Controller/MainTabBarController.swift (hunk 1 above)
Comment (dokun1) [reply iOS/rainbow/Controller/MainTabBarController.swift]: The key element I want to express is that the game can still be played. @anamita what do you think?

Comment (sanjeevghimire) [reply iOS/rainbow/Controller/MainTabBarController.swift]: That would be when someone taps `view leaderboard`

Comment (sanjeevghimire) [reply iOS/rainbow/Controller/MainTabBarController.swift]: this happens when you start the game and end the game. May be we could say " there is error but keep playing"

Comment (dokun1) [reply iOS/rainbow/Controller/MainTabBarController.swift]: What about "Couldn't connect to the server - keep playing!"

**Pull request merged**